node/jaeger/src/lib.rs

```diff
@@ -112,6 +112,7 @@ impl Jaeger {
 		// Spawn a background task that pulls span information and sends them on the network.
 		spawner.spawn(
 			"jaeger-collector",
+			"jaeger",
```

Comment:
:+1:

Reply:
Note that the casing is slightly inconsistent here, as @ordian already mentioned.